pyproject.toml

```diff
@@ -71,6 +71,26 @@ build.hooks.vcs.version-file = "src/platformdirs/version.py"
 build.targets.sdist.include = ["/src", "/tests", "/tox.ini"]
 version.source = "vcs"
 
+[tool.hatch.envs.default]
+features = ["docs", "test"]
+
+[tool.hatch.envs.default.scripts]
+test = """\
+  pytest --cov platformdirs --cov tests \
+  --cov-config=pyproject.toml --no-cov-on-fail \
+  --cov-report term-missing:skip-covered --cov-context=test \
+  --cov-report html:htmlcov \
+  tests
+  """
+check-types = [
+  "mypy --strict src",
```

Comment:
strict is already set in the mypy config
i believe a single command passing all paths is better here

Reply:
Sure, I just copied what I saw in `tox.ini`. Would you like me to change it there, as well?